### PR TITLE
Move test util functions to tpkg

### PR DESCRIPTION
**Pull request by muXxer**
We have several occurrences of test functions in our code that do exactly the same.
This PR removes duplicated code by moving the test utility functions to a new package called `tpkg` (like in iota.go).

To be merged after #1593 

Attention: Set the base branch to develop before merging!

### CODE REVIEW

File: pkg/testsuite/utils.go [hunk 1]
```diff
@@ -105,8 +105,7 @@ func (b *BlockBuilder) fromWalletOutputs() ([]*utxo.Output, uint64) {
 	} else {
 		if b.fakeInputs {
 			// Add a fake output with enough balance to create a valid transaction
-			fakeInputID := iotago.OutputID{}
-			copy(fakeInputID[:], randBytes(iotago.TransactionIDLength))
+			fakeInputID := tpkg.RandOutputID()
```
Comment (alexsporn) [inline pkg/testsuite/utils.go:108]: Better duplicate and create a RandTransactionID, because if anything changes then this will break

Comment (alexsporn) [reply pkg/testsuite/utils.go:108]: There is even one already `RandTransactionID`

Comment (muXxer) [reply pkg/testsuite/utils.go:108]: But `RandOutputID` is already correct in this context, no?

Comment (alexsporn) [reply pkg/testsuite/utils.go:108]: I read BlockID :weirdsmileguy:

**Review by alexsporn — APPROVED** [review]

**Pull request merged**